docs/developers/references/traits/autoscale.md

```diff
@@ -0,0 +1,100 @@
+  App helloworld deployed
+  ```
+  
+  Check the replicas of Deployment `frontend` which is deployed by workload webservice `helloworld` and there is one replica.
+  
+  (TODO: The command below needs to be replaced  with `vela show` to check the replicas.)
+  ```
+  $ kubectl get deploy frontend
+  NAME       READY   UP-TO-DATE   AVAILABLE   AGE
+  frontend   1/1     1            1           2m52s
+  ```
+
+- Scale the application by `cron`
+  ```
+  $ vela autoscale helloworld --svc frontend --minReplicas 1 --maxReplicas 4 --replicas 2 --name cron-test --startAt 21:00 --duration 2h --days "Monday, Tuesday"
+  Adding autoscale for app frontend
+  ⠋ Deploying ...
+  ✅ Application Deployed Successfully!
+    - Name: frontend
+      Type: webservice
+      HEALTHY Ready: 1/1
+      Last Deployment:
+        Created at: 2020-11-03 20:53:50 +0800 CST
+        Updated at: 2020-11-03T21:01:20+08:00
+      Traits:
+        - autoscale: 
+            maxReplicas=4
+            minReplicas=1
+            replicas=2
+            startAt=21:00
+            timezone=Asia/Shanghai
+            days=Monday, Tuesday
+            name=cron-test
+            type=cron
+            duration=2h
+  ```
+  
+  The time is `21:07` which is in the active period of the trait which started at `21:00` and the duration is two hours.
+  Check the replicas of Deployment `frontend` again, it has been scaled to 2.
+  ```
+  $ kubectl get deploy
+  NAME       READY   UP-TO-DATE   AVAILABLE   AGE
+  frontend   2/2     2            2           8m42s
+  ```
+  
+  Wait after the period ends, the replicas will be one eventually.
+
+## Debug
+  This part is only for platform builders. If you are an application developer, just skip it.
+  
+  - Scaling doesn't take effect
+  
+    This might be caused by the unreadiness of KEDA ScaledObject which is deployed by Autoscale trait.
+    
+    ```
+    $ kubectl get scaledobject.keda.sh
+    NAME           SCALETARGETKIND   SCALETARGETNAME    TRIGGERS   AUTHENTICATION   READY   ACTIVE   AGE
+    trait-scaler                     component-scaler   cron                                         3m49s
+    ```
+    
+    Please check those Pods of Keda.
+    ```
+    $ kubectl get pods -n keda
+    NAME                                               READY   STATUS             RESTARTS   AGE
+    keda-operator-6d89f67964-smksp                     0/1     CrashLoopBackOff   8          22m
+    keda-operator-metrics-apiserver-77598644dd-w7th5   1/1     Running            0          22m
+    ```
+    
+    Try to fix the issue, and those Pods will become `READY`.
+    ```
+    k get pods -n keda
+    NAME                                               READY   STATUS    RESTARTS   AGE
+    keda-operator-695d978ddb-w5qct                     1/1     Running   0          7m53s
+    keda-operator-metrics-apiserver-77598644dd-f4fjq   1/1     Running   0          7m53s
+    ```    
```

Comment:
Don't add the debug information here, please remove them.

Remember who are reading your docs.

Reply:
Though I said `This part is only for platform builders. If you are an application developer, just skip it.`, but, as you said, this is for app developer. Let me move it to somewhere for platform builders. I'd like to add some more debug cases as I hit those issues and they waste me much time as a platform builder.